src/App.java

```diff
@@ -1,5 +1,56 @@
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.util.*;
+
 public class App {
-    public static void main(String[] args) throws Exception {
-        System.out.println("Hello, World!");
+    /*loads the flashcards from flashcard.txt*/

```

Comment:
Documentation comments should use two stars at the beginning, that's the special signal to VS Code that they are documentation. So this could be

```java
/** loads the flashcards from flashcard.txt */
```